frontend/src/api/author_api.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
```

Comment:
I already wrote this in post_display_api.js so I can remove it from there and we can have them in separate files, probably more visible